File: typescript-client-sdk/src/sdk/models/shared/emailupdate.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class EmailUpdate extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=additional" })
  additional?: Record<string, any>;

  @SpeakeasyMetadata({ data: "json, name=isIncoming" })
  isIncoming?: boolean;

  @SpeakeasyMetadata({ data: "json, name=ownerUserId" })
  ownerUserId?: string;

  @SpeakeasyMetadata({ data: "json, name=status" })
  status?: string;
}

File: go-client-sdk/pkg/models/shared/emailupdate.go
package shared

type EmailUpdate struct {
	Additional  map[string]interface{} `json:"additional,omitempty"`
	IsIncoming  *bool                  `json:"isIncoming,omitempty"`
	OwnerUserID *string                `json:"ownerUserId,omitempty"`
	Status      *string                `json:"status,omitempty"`
}
